python/poetry.lock

```diff
@@ -909,7 +875,6 @@ version = "1.1.1"
 description = "PPMd compression/decompression library"
 optional = false
 python-versions = ">=3.9"
-groups = ["tutorials"]
```

Comment:
why do you remove this group? 
I don't think it should have been in the python dir.. I think it should have been in a doc dir.. but that's another PR..

Reply:
It is automatically generated by `poetry lock` and is relying on the current structure of `pyproject.toml`